components/Member/Board/MemberCard/MemberCard.tsx

```diff
@@ -92,9 +92,11 @@ function MemberCard({
             onMouseEnter={handleMouseEnterTags}
             onMouseLeave={handleMouseLeaveTags}
           >
-            <div className={cx("tag")}>
-              <span className={cx("tagName")}>{generation}기</span>
-            </div>
+            {generation === null ? null : (
```

Comment:
```suggestion
            {generation === undefined ? null : (
```

타입상 이게 나을듯? optional 프로퍼티는 undefined로 처리되니께

Reply:
일단 json파일에는 내가 직접 명시적으로 null이라고 적어놔서 여기도 null이랑 비교해야되는데, 그냥 ==으로 비교할까?

Reply:
흠 그냥 바꾸지말고 나중에 서버랑 연결할 때 문제되면 다시 작업하겠슴다

Reply:
아 그래?? 그럼 위에 타입을
{ generation?: string} 말고
{ generation: string | null } 로 하는게 나을듯

Reply:
엇 이미 머지해버림 ㅋㅋㅋㅋㅋ

Reply:
아 ? 의 의미가 nullable하다는 게 아니고 undefined 일 수 있다는 거였어?

Reply:
아 ㅇㅇ 자스에서 없는 프로퍼티 읽으면 null이 아니라 undefined가 떠서 타입스크립트도 string | undefined로 타입체킹함ㅋㅋㅋ

Reply:
요건 제가 고칠게여~

Reply:
ㅇㅎ 또 하나 배웠다!
감사감사용